src/writer.rs

```diff
@@ -63,11 +63,25 @@ impl<W: Write> BzCompressor<W> {
         }
         Ok(self.w.take().unwrap())
     }
+
+    /// Returns the number of bytes produced by the compressor
+    pub fn total_out(&self) -> u64 {
+        self.stream.total_out()
+    }
+
+    /// Returns the number of bytes consumed by the compressor
```

Comment:
Could you add a note here saying that it may differ from the amount of bytes successfully written to the compressor as some data may still be buffered?